Clarify the use of #define USE_IBEACON_ESP32

pearson opened the pull request:
## Description:

A fix to the comment text for `#define USE_IBEACON_ESP32` and `#define USE_IBEACON`:
- There was no description for `#define USE_IBEACON_ESP32`, so text was added to clarify it is the define to use for the ESP32 internal Bluetooth radio
- Indentation was added to `#define USE_IBEACON_ESP32` and `#define USE_IBEACON` to clarify that the `USE_BLE_ESP32` define is required.

## Checklist:
  - [X] The pull request is done against the latest development branch
  - [X] Only relevant files were touched
  - [X] Only one feature/fix was added per PR and the code change compiles without warnings
  - [X] The code change is tested and works with Tasmota core ESP8266 V.2.7.4.9
  - [X] The code change is tested and works with Tasmota core ESP32 V.2.0.3
  - [X] I accept the [CLA](https://github.com/arendst/Tasmota/blob/development/CONTRIBUTING.md#contributor-license-agreement-cla).

_NOTE: The code change must pass CI tests. **Your PR cannot be merged unless tests pass**_

pearson committed:
Clarify the use of #define USE_IBEACON_ESP32

A fix to the comment text for `#define USE_IBEACON_ESP32` and `#define
USE_IBEACON`:
- There was no description for `#define USE_IBEACON_ESP32`, so text was
  added to clarify it is the define to use for the ESP32 internal
  Bluetooth radio
  - Indentation was added to `#define USE_IBEACON_ESP32` and `#define
    USE_IBEACON` to clarify that the `USE_BLE_ESP32` define is required.

ascillato commented:
Hi, I think that it is missing a commit. Your description does not match the code modification. This PR is just enabling those features for all versions. Can you recheck it?

pearson commented:
@ascillato , you are correct. Somehow I managed to commit the wrong file. Sorry about that -- I'll fix it shortly.

pearson committed:
Clarify the use of #define USE_IBEACON_ESP32

A fix to the comment text for `#define USE_IBEACON_ESP32` and `#define
USE_IBEACON`:
- There was no description for `#define USE_IBEACON_ESP32`, so text was
  added to clarify it is the define to use for the ESP32 internal
  Bluetooth radio
  - Indentation was added to `#define USE_IBEACON_ESP32` and `#define
    USE_IBEACON` to clarify that the `USE_BLE_ESP32` define is required.

pearson committed:
Clarify the use of #define USE_IBEACON_ESP32

A fix to the comment text for `#define USE_IBEACON_ESP32` and `#define
USE_IBEACON`:
- There was no description for `#define USE_IBEACON_ESP32`, so text was
  added to clarify it is the define to use for the ESP32 internal
  Bluetooth radio
  - Indentation was added to `#define USE_IBEACON_ESP32` and `#define
    USE_IBEACON` to clarify that the `USE_BLE_ESP32` define is required.

pearson commented:
@ascillato I don't use git as much as I should... That should have been an amend, not two separate commits.
